.github/workflows/ci.yml

```
@@ -145,15 +145,67 @@ jobs:
         run: |
           npm install 
           npx rolldown -v
-        working-directory: test/package
+        working-directory: test/package/rolldown
 
       - name: "Test package"
-        run: npm run test:package -- -t rolldown
+        run: npm run test:package:rolldown
+
+  test-package-vite:
+    name: "vite v${{ vite.rollup }} / node ${{ matrix.node }} / ${{ matrix.os }}"
```

actionlint (1.7.7)

undefined variable "vite". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy", "vars"

(expression)